doc/user/layouts/shortcodes/network-security/privatelink-kafka.md

```diff
@@ -97,6 +97,10 @@ and retrieve the AWS principal needed to configure the AWS PrivateLink service.
       You **do not need** to specify `AVAILABILITY ZONES` manually — these will
       be optimally auto-assigned when none are provided.
 
+    - For Kafka connections, it is required for [cross-zone load balancing](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html) to be
```

Comment:
trivial ... Optionally,  you can make this into a note above line 82 to make it more prominent.  But ... will leave up to you as I'm not sure how prominent you want to make it.